support-frontend/assets/pages/contributions-landing/components/ContributionForm.jsx

```diff
@@ -295,6 +317,20 @@ function withProps(props: PropTypes) {
         <ContributionFormFields />
         <PaymentMethodSelector />
 
+        {
+          props.paymentMethod === Sepa && (
+            <SepaForm
+              iban={props.sepaData.iban}
+              accountHolderName={props.sepaData.accountHolderName}
+              accountHolderConfirmation={props.sepaData.accountHolderConfirmation}
+              updateIban={props.setSepaIban}
+              updateAccountHolderName={props.setSepaAccountHolderName}
+              updateAccountHolderConfirmation={props.setSepaAccountHolderConfirmation}
+              checkoutFormHasBeenSubmitted={props.checkoutFormHasBeenSubmitted}
```

Comment:
Does React / JSX not have a macro which can automatically create attributes from all the properties of an object to save you having to do all this boilerplate?